starter-code/src/App.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Yay from "./yay";
+import "./Style.css"
+import Wallouh from "./wallouh";
+import Lolilol from "./lolilol";
+
+// import Greetings from "./Greetings"
+
+export default function App() {
+    return (
+        <div className="mainBox">
+        <Lolilol />
```

Comment:
This component is supposed to receive an image props but it's never going down
Nice component name though.